buildSrc/src/main/kotlin/assertk.multiplatform.gradle.kts

```diff
@@ -27,19 +31,10 @@ if (libs.versions.assertk.get().endsWith("SNAPSHOT")) {
     }
 }
 
-val nativeTargets = arrayOf(
-    "linuxX64", "linuxArm64",
-    "macosX64", "macosArm64",
-    "mingwX64",
-    "iosArm32", "iosArm64", "iosX64", "iosSimulatorArm64",
-    "tvosArm64", "tvosX64", "tvosSimulatorArm64",
-    "watchosArm32", "watchosArm64", "watchosX86", "watchosX64", "watchosSimulatorArm64", "watchosDeviceArm64",
-    "androidNativeArm32", "androidNativeArm64", "androidNativeX86", "androidNativeX64",
-)
-
 kotlin {
     jvm()
     js {
+        browser()
         nodejs()
```

Comment:
Why test this and the WASM target twice?

Reply:
Aren't all of them needed for consumers? Especially if https://youtrack.jetbrains.com/issue/KT-47038 is the direction that KMP is headed with this.

Reply:
This does not affect consumers in any way today. If the targets split they will split as WASM has, not using these functions. Today it primarily dictates where tests run, and for this library there's no point in running the tests in a browser.

Reply:
It doesn't affect the generated json? Or is that handled at the consumer side?

Reply:
Yeah JS is only generated at the very end, by the consumer.